tests/mock_tables/dbconnector.py

```diff
@@ -133,12 +134,22 @@ def keys(self, pattern='*'):
         # Find every key that matches the pattern
         return [key for key in self.redis.keys() if regex.match(key)]
 
+def _isInit():
+    return   SonicDBConfig._sonic_db_config_init
```

Comment:
If we have this function implemented in swsssdk on par with swsscommon, will it be better than implement in and every mock class?

Reply:
Raised a PR to add these functions swsssdk https://github.com/Azure/sonic-py-swsssdk/pull/107